app/services/idv/steps/doc_auth_base_step.rb

```diff
@@ -65,7 +65,9 @@ def extract_pii_from_doc(response, store_in_session: false)
         )
 
         flow_session[:had_barcode_read_failure] = response.attention_with_barcode?
-        flow_session[:pii_from_doc] = pii_from_doc if store_in_session
+        if store_in_session
+          flow_session[:pii_from_doc] = flow_session[:pii_from_doc].to_h.merge(pii_from_doc)
+        end
```

Comment:
Without these changes, the document capture PII extraction will wipe out the previously-entered SSN.

I _think_ it should be fine, but a couple alternatives:

1. Cherry-pick the individual field(s) we want to preserve (`:ssn`)
2. Update to avoid [storing SSN in the `pii_from_doc` hash](https://github.com/18F/identity-idp/blob/5d9f30c2df5f021a222093851f583240799df0fa/app/services/idv/steps/ssn_step.rb#L9)

Reply:
I like option 2 but that seems much bigger than this PR, so I would vote to leave this PR as-is and track moving non-doc PII out of `pii_from_doc` in a separate ticket

Reply:
I like option 2 as well. I am also hopeful that one day in the glorious future it'll be easier once we start breaking down the FSM